src/main/java/frc/robot/Robot.java

```diff
@@ -24,32 +19,55 @@ public class Robot extends TimedRobot {
     private ShooterSystem shooterSystem;
     private IntakeSystem intakeSystem;
     private ArmSystem armSystem;
-    private XboxController xboxController;
+    private XboxController driveController;
+    private XboxController operatorController;
+    private ParallelCommandGroup cancelAllCommands;
+    private ParallelCommandGroup shootToAmp;
 
     @Override
     public void robotInit() {
         driveSubsystem = new DriveSubsystem();
         shooterSystem = new ShooterSystem();
         intakeSystem = new IntakeSystem();
         armSystem = new ArmSystem();
-        xboxController = new XboxController(0);
-
-        POVButton dPadUp = new POVButton(xboxController, 0);
-        POVButton dPadDown = new POVButton(xboxController, 180);
+        driveController = new XboxController(0);
+        operatorController = new XboxController(1);
+
+        JoystickButton Ybutton = new JoystickButton(operatorController, XboxController.Button.kY.value);
+        //shooterSystem.getCurrentCommand().cancel();
+
+        ShooterPID shooterCommand = new ShooterPID(shooterSystem, 5000);
+        ArmMoveToShooterCommand armCommand = new ArmMoveToShooterCommand(armSystem);
+
+        shootToAmp = new ParallelCommandGroup(
```

Comment:
so basically: move arm to 130 degrees, push note out and done?

Reply:
yes, isnt it?